.firebaserc

```diff
@@ -1,5 +1,5 @@
 {
   "projects": {
-    "default": "devfestpescara-2023"
+    "default": "hedwig-demo"
```

Comment:
Sicuri che vogliamo lasciare questo default?

Reply:
Better add the .firebaserc to the gitignore?

Reply:
mmmh dunno, is it something required I guess... ok, maybe we could leave it there, up to you in this case 😆